tutorials/XGI in 5 minutes.ipynb

```diff
@@ -0,0 +1,477 @@
+{
```

Comment:
<p>maybe add a link to where people can find out about other generators</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>